packages/cmr-client/package.json

```diff
@@ -35,9 +35,12 @@
   "dependencies": {
     "@cumulus/aws-client": "2.0.1",
     "@cumulus/logger": "2.0.1",
-    "got": "^9.6.0",
+    "got": "^11.7.0",
     "lodash": "^4.17.15",
     "public-ip": "^3.0.0",
     "xml2js": "^0.4.19"
+  },
+  "devDependencies": {
```

Comment:
Tangential question: why are these dev dependencies not moved to the top level?

Reply:
I wanted to make sure that the version of the type matched the version of the package, but I guess it could probably be moved up.